Doc/whatsnew/3.14.rst

```diff
@@ -301,11 +301,11 @@ For further information on how to build Python, see
     cautiously revised down to 3-5%. While we expect performance results to be better
```

Comment:
maybe a silly question, why do we expect performance numbers to be better than what we report?

Reply:
The computed goto bug in theory shouldnt affect modern systems that much, but it's unclear for now so I just put that therr.